lang/nl.json

```diff
@@ -0,0 +1,95 @@
+    "Change e-mail address": "E-mailadres wijzigen",
+    "Change password": "Wachtwoord wijzigen",
+    "Changed successfully!": "Succesvol gewijzigd!",
+    "Contact information": "Contactgegevens",
+    "Continue shopping": "Verder winkelen",
+    "Coupon code": "Kortingscode",
+    "Current password": "Huidig wachtwoord",
+    "Customer centre": "Klantenservice",
+    "Default billing address": "Standaard factuuradres",
+    "Default shipping address": "Standaard verzendadres",
+    "Delete address": "Adres verwijderen",
+    "Delivery method": "Bezorgmethode",
+    "Deselect as billing": "Deselecteer als factuuradres",
+    "Deselect as shipping": "Deselecteer als verzendadres",
+    "Don't have an account yet? Create an account and enjoy faster ordering, repeat orders, status of your order, easy returns and more!": "Nog geen account? Maak een account aan en geniet van sneller bestellen, herhaalde bestellingen, bestelstatus, gemakkelijke retouren en meer!",
+    "Edit": "Bewerken",
+    "Email": "E-mail",
+    "Enter your e-mail address": "Voer uw e-mailadres in",
+    "Enter your email address": "Voer uw e-mailadres in",
+    "Enter your email address below, you will receive an email within minutes to reset the password.": "Voer hieronder uw e-mailadres in, u ontvangt binnen enkele minuten een e-mail om het wachtwoord te resetten.",
+    "Everything centralized in 1 place such as returns, exchanges and customer service": "Alles gecentraliseerd op 1 plek zoals retouren, ruilingen en klantenservice",
+    "Free": "Gratis",
+    "Go to your account": "Ga naar uw account",
+    "Is this not your account?": "Is dit niet uw account?",
+    "Log out": "Uitloggen",
+    "Logout from your account": "Uitloggen van uw account",
+    "My addresses": "Mijn adressen",
+    "New password": "Nieuw wachtwoord",
+    "No account yet? Create an account and benefit instantly from repeat orders, order statuses and easy returns!": "Nog geen account? Maak direct een account aan en profiteer van herhaalde bestellingen, bestelstatus en gemakkelijke retouren!",
+    "No default address has been set yet.": "Er is nog geen standaardadres ingesteld.",
+    "Number of products": "Aantal producten",
+    "Order": "Bestelling",
+    "Order again": "Bestelling herhalen",
+    "Order date": "Besteldatum",
+    "Order overview": "Bestellingsoverzicht",
+    "Ordered within 2 minutes": "Besteld binnen 2 minuten",
+    "Place repeat order / View orders": "Herhaal bestelling / Bekijk bestellingen",
+    "Please select an address": "Selecteer een adres",
+    "Price": "Prijs",
+    "Product": "Product",
+    "Quick and easy ordering without having to fill in your details every time": "Snel en eenvoudig bestellen zonder elke keer uw gegevens in te vullen",
+    "Register": "Registreren",
+    "Register account": "Account registreren",
+    "Register within 1 minute": "Registreer binnen 1 minuut",
```

Comment:
This is a USP, correct?

Reply:
Yup, however one that does exist in the source: https://github.com/rapidez/checkout-theme/blob/master/resources/views/account/partials/register.blade.php#L3